db/db.pl

```diff
@@ -2292,6 +2307,9 @@ sub dataNodes
         sessions2Update();
         checkForOld5Indices();
         usersUpdate();
+    } elsif ($main::versionNumber <= 56) {
+        usersUpdate();
+        queriesUpdate();
```

Comment:
You need to add queriesUpdate for all the other if cases that don't have one already, also every if should have the sessions2Update() so instead of making a new section probably just change the if from <= 55 to <= 56